src/vcBPA.cpp

```diff
@@ -1014,16 +1055,19 @@ void vcBPA_ConvertClose(vdkConvertCustomItem *pConvertInput)
 {
   vcBPAConvertItem *pBPA = (vcBPAConvertItem *)pConvertInput->pData;
   pBPA->running = false;
+
   udThread_Join(pBPA->pThread);
   udThread_Destroy(&pBPA->pThread);
+
   pBPA->activeItem.oldGrid.Deinit();
   pBPA->activeItem = {};
 
-  vcBPAConvertItemData itemData;
-  while (udSafeDeque_PopFront(pBPA->pQueueItems, &itemData) == udR_Success)
-    itemData.oldGrid.Deinit();
-  udSafeDeque_Destroy(&pBPA->pQueueItems);
+  vcBPAOctNode node;
+  while (udSafeDeque_PopFront(pBPA->pLoadList, &node) == udR_Success)
+    udFree(node.pChildren);
```

Comment:
I don't believe that `pLoadList` contains all of the nodes at the end of execution? Above you call `PopBack` but you don't push that node back on and then the scope ends. I would recommend that you use the `rootNode` instead.

Reply:
It doesn't but close can be called partway through the process

Reply:
My point is that you're only cleaning up the nodes that are in the `pLoadList` - which isn't all of them. This is a memory leak that needs to be fixed.